### DISCUSSION TITLE
Windows support

### DISCUSSION THREAD

**Discussion by andreprawira** [aws/codewhisperer-command-line-discussions#17] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description:
Could we please have a windows support for windows 10 and 11?

**Comment by brendanfalk** [comment]
Dupe of https://github.com/aws/codewhisperer-command-line-discussions/discussions/15
